semcore/date-picker/src/components/Calendar.jsx

```diff
@@ -329,13 +343,15 @@ class CalendarDaysRoot extends CalendarAbstract {
     const description = this.describeValue();
 
     return sstyled(styles)(
-      <SCalendar render={Box}>
-        <CalendarWeekDays locale={locale} />
-        <SGridDays onMouseLeave={this.handleMouseLeave}>
+      <SCalendar render={Box} {...this.getRootProps()}>
+        <CalendarWeekDays locale={locale} role='row' />
```

Comment:
why do we need `row` here? Should we `walk` through the header of Calendar?

Reply:
It's required to put row between grid and cell